docs/dev-environment/setting-up-dev-environment.md

```diff
@@ -30,9 +30,9 @@ npm install
 
 # Setup Database
 
-You should first install MariaDB, which for Linux can be done here: https://downloads.mariadb.org/mariadb/repositories, and a guide for MacOS is here: https://mariadb.com/kb/en/library/installing-mariadb-on-macos-using-homebrew/. We recommend using either a Linux distribution or MacOS as your operating system when developing for The Gazelle. We use version 10.2 of MariaDB in production / CircleCI so that's the recommended version. If you are only setting up for development and not for deployment you can also use MySQL which is 100% compatible.
+You should first install MariaDB, which for Linux can be done here: https://downloads.mariadb.org/mariadb/repositories, and a guide for MacOS is here: https://mariadb.com/kb/en/library/installing-mariadb-on-macos-using-homebrew/. Follow the installation steps found on the guide, except for the last step: 'sudo apt install mariadb-server'. Run 'sudo apt install mariadb-server-10.2' instead as we use version 10.2 of MariaDB in production / CircleCI so that's the recommended version. We recommend using either a Linux distribution or MacOS as your operating system when developing for The Gazelle. If you are only setting up for development and not for deployment you can also use MySQL which is 100% compatible.
```

Comment:
Can you update this to mention specifically that the command - 'sudo apt install mariadb-server-10.2' should be run on Linux only?
Additionally, always include commands inside backticks (`) - `inside backticks` - for the formatting of course :)